### ISSUE TITLE
Add layers before and after pretrained model

### ISSUE THREAD

**Issue by konstantin-frolov** [keras-team/keras#14280]
Hi.
I want to create my model with custom layers before and after pretrained model and see all layers in summary, for extracting gradients.
For example if use:
```
inputs = keras.layers.Input(input_shape=(320, 320,1))
x = keras.layers.Conv2D(3,3, padding='valid')(inputs) #change channels from 1 to 3
x = base_model(x)
x = keras.layers.GlobalAveragePooling2D()(x)
outputs = keras.layers.Dense(num_classes, activation='sigmoid')
model = Model(inputs, outputs)
```
In model.summary we have 5 layers, when base_model it's only one layer.
I want see all layers of base model in summary.

If I use
```
x = base_model.output
x = keras.layers.GlobalAveragePooling2D()(x)
outputs = keras.layers.Dense(num_classes, activation='sigmoid')
model = Model(base_model.input, outputs)
```
In model.summary we have all layers of base_model. But I don't understand how add layers before base model.

**Comment by sushreebarsa** [comment]
@konstantin-frolov  Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
